mfhdf/libsrc/file.c

```diff
@@ -501,23 +501,14 @@ ncnobuf(int cdfid)
 static char *
 NCtempname(const char *proto)
 {
-/* NO_ACCESS defined if the OS lacks the access() function */
-#ifndef NO_ACCESS
-#define TN_NACCES 1
-#else
-#define TN_NACCES 0
-#endif /* !NO_ACCESS */
-/* NO_GETPID defined if the OS lacks the getpid() function */
-#ifndef NO_GETPID
+#define TN_NACCES  1
 #define TN_NDIGITS 4
 #if defined H4_HAVE_WIN32_API
     typedef int pid_t;
+#else
+    pid_t getpid(void);
```

Comment:
It looks like you are defining a new prototype for getpid() on non-Windows systems? I don't think that's going to be the right thing to do.

Reply:
I think it's trying to match the prototypes.  Windows doesn't define pid_t so that is being set to int. Mingw does define pid_t but getpid returns int. *nix just use pid_t.